pos_hide_empty_category/views/assets.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2021 - Today: GRAP (http://www.grap.coop)
+    @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+    <template id="assets_frontend" inherit_id="point_of_sale.assets">
+        <xpath expr="." position="inside">
```

Comment:
```suggestion
        <xpath expr="//script[last()]" position="after">
```
So that we OCD people are happy ;D

Reply:
I commited your suggestion, but didn't understood this one. Could you help me understand why it is better ? (And what is OCD ?)

thanks !

Reply:
OCD as in Obsessive compulsive disorder.. 
bad joke about liking orderliness

Technically no benefit afaik, it's just to keep scripts all together, and not mixed with stylesheets.
It's a common practice in web development to load css before js (so that the page "seems" to load faster).. but in odoo I'm not sure it makes any difference